src/main/scala/com/aamend/spark/gdelt/GdeltParser.scala

```diff
@@ -64,7 +150,7 @@ object GdeltParser {
       adm1Code = T(()=>tokens(38)),
       adm2Code = T(()=>tokens(39)),
       geoPoint = T(()=>actor1GeoPoint),
-      featureId = T(()=>tokens(40))
+      featureId = T(()=>tokens(42))
```

Comment:
Is that expected? I've never used feature ID in any of my analysis, so may have missed that bug - if a bug

Reply:
That's a bug. As the values differ.

Reply:
In the old version the token(40) appears twice, first as latitude and then as feature Id. According to the documentation feature id should be on the 43rd column which corresponds to token(42).